### PR TITLE
added instructions for extracting bounding boxes

### PR DESCRIPTION
**Pull request by AkashParua**
Added in-depth instructions in YOLONAS.md file on how to get the bounding box(s) of detected objects.
Wrote a detailed explanation of "output._images_prediction_lst" , where  output=mode.predict(<src>) .

### CODE REVIEW

File: YOLONAS.md [hunk 1]
```diff
@@ -32,6 +32,21 @@ import super_gradients
...
+```
+## Extract bounding boxes 
+```python
+
+output = yolo_nas.predict("https://deci-pretrained-models.s3.amazonaws.com/sample_images/beatles-abbeyroad.jpg")
+'''
+output._images_prediction_lst = [ImageDetectionPrediction(image=array([[[.,.,.],[.,.,.],[.,.,.]...],...],dtype=float32),
+                                prediction=DetectionPrediciton(bboxes_xyxy=array([],shape(0,4),dtype=float32),confidence=array([],dtype=float32),labels=array([],dtype=float32)),class_names=['name','of','all','classes'])]
+'''
+predicton_lst = list(output._images_prediction_lst)[0]
```
Comment (Louis-Dupont) [inline YOLONAS.md:44]: Since `3.1.2` you can run
`image_prediction = output[0]`

File: YOLONAS.md [hunk 2]
```diff
@@ -32,6 +32,21 @@ import super_gradients
...
+bboxes = prediction_lst.prediction.bboxes_xyxy #bboxes = [[Xmin,Ymin,Xmax,Ymax],..] list of all annotation(s) for detected object(s) 
+class_names = prediction_lst.class_names = ['name','of','all','classes'] #all the class name(s)
+class_name_indexes = prediction_lst.prediction.labels.astype(int) #index of each detected object in class_names(corresponding to each bounding box)
+confidences =  prediction_lst.prediction.confidence.astype(float) #confidence value(s) in float for each bounding boxes
```
Comment (Louis-Dupont) [inline YOLONAS.md:48]: I think we can format it slighltly differently to improve readability


```
prediction = output[0].prediction

bboxes = prediction.bboxes_xyxy                     # [[Xmin,Ymin,Xmax,Ymax],..] list of all annotation(s) for detected object(s) 
class_names = prediction.class_names.               # ['Class1', 'Class2', ...] List of the class names
class_name_indexes = prediction.labels.astype(int)  # [2, 3, 1, 1, 2, ....] Index of each detected object in class_names(corresponding to each bounding box)
confidences =  prediction.confidence.astype(float)  # [0.3, 0.1, 0.9, ...] Confidence value(s) in float for each bounding boxes
```

File: YOLONAS.md [hunk 3]
```diff
@@ -32,6 +32,21 @@ import super_gradients
...
 yolo_nas.predict("https://deci-pretrained-models.s3.amazonaws.com/sample_images/beatles-abbeyroad.jpg").show()
+```
+## Extract bounding boxes 
+```python
+
+output = yolo_nas.predict("https://deci-pretrained-models.s3.amazonaws.com/sample_images/beatles-abbeyroad.jpg")
+'''
+output._images_prediction_lst = [ImageDetectionPrediction(image=array([[[.,.,.],[.,.,.],[.,.,.]...],...],dtype=float32),
+                                prediction=DetectionPrediciton(bboxes_xyxy=array([],shape(0,4),dtype=float32),confidence=array([],dtype=float32),labels=array([],dtype=float32)),class_names=['name','of','all','classes'])]
+'''
```
Comment (Louis-Dupont) [inline YOLONAS.md:43]: I believe that with the example on how to get each attribute that you added below, this is redundant so I think it's better to only keep the example, and remove this string from the tutorial

**Review by Louis-Dupont — COMMENTED** [review]
I left some comments in-line, thanks for the contribution!

File: YOLONAS.md (hunk 2 above)
Comment (AkashParua) [reply YOLONAS.md]: Thanks ! I am new to open source , really appreciate the guidance

File: YOLONAS.md [hunk 4]
```diff
@@ -27,11 +27,13 @@ YOLO-NAS's architecture employs quantization-aware blocks and selective quantiza
...
 ```python
-import super_gradients
-
-yolo_nas = super_gradients.training.models.get("yolo_nas_l", pretrained_weights="coco").cuda()
-yolo_nas.predict("https://deci-pretrained-models.s3.amazonaws.com/sample_images/beatles-abbeyroad.jpg").show()
+prediction = output[0].prediction
+bboxes = prediction.bboxes_xyxy                     # [[Xmin,Ymin,Xmax,Ymax],..] list of all annotation(s) for detected object(s) 
+class_names = prediction.class_names.               # ['Class1', 'Class2', ...] List of the class names
+class_name_indexes = prediction.labels.astype(int)  # [2, 3, 1, 1, 2, ....] Index of each detected object in class_names(corresponding to each bounding box)
+confidences =  prediction.confidence.astype(float)  # [0.3, 0.1, 0.9, ...] Confidence value(s) in float for each bounding boxes
```
Comment (Louis-Dupont) [inline YOLONAS.md:36]: @AkashParua 
Looks like the `yolonas_predict` part was also removed, maybe by mistake. Let's put it back
```python
yolo_nas = super_gradients.training.models.get("yolo_nas_l", pretrained_weights="coco").cuda()
model_predictions  = yolo_nas.predict("https://deci-pretrained-models.s3.amazonaws.com/sample_images/beatles-abbeyroad.jpg").show()

prediction = model_predictions[0].prediction        # One prediction per image - Here we work with 1 image so we get the first.

bboxes = prediction.bboxes_xyxy                     # [[Xmin,Ymin,Xmax,Ymax],..] list of all annotation(s) for detected object(s) 
bboxes = prediction.bboxes_xyxy                     # [[Xmin,Ymin,Xmax,Ymax],..] list of all annotation(s) for detected object(s) 
class_names = prediction.class_names.               # ['Class1', 'Class2', ...] List of the class names
class_name_indexes = prediction.labels.astype(int)  # [2, 3, 1, 1, 2, ....] Index of each detected object in class_names(corresponding to each bounding box)
confidences =  prediction.confidence.astype(float)  # [0.3, 0.1, 0.9, ...] Confidence value(s) in float for each bounding boxes
```

Then, LGTM 👍

Comment (AkashParua) [reply YOLONAS.md]: Done !

**Review by Louis-Dupont — APPROVED** [review]
LGTM! Thanks for contribution 🚀

**Pull request merged**